pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pymedphys"
-version = "0.34.0-dev6"
+version = "0.34.0-dev8"
```

Comment:
are we past dev8 now?
up to dev10 maybe?

Reply:
It was changing up the stack... so once this stack merges it'll say dev9, but yes, should a new dev release happen it'd need to be dev10